### PR TITLE
Tahoe style certs for PennState

### PR DESCRIPTION
**Pull request by bryanlandia**

### CODE REVIEW

**Comment by bryanlandia** [comment]
Updated to rebase off of unmerged amir/ginkgo.psu/BLACK-700 since @amirtds and I are both working on changes to push to staging

File: lms/templates/theme-variables.html [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
 <%! from openedx.core.djangoapps.site_configuration.models import SiteConfiguration %>
 <%! from django.contrib.staticfiles.templatetags.staticfiles import static %>
-<%! from datetime import datetime %>
+<%! from django.utils.encoding import force_text %>
+<%! from django.utils import translation %>
+<%! from django.utils.translation import ugettext, ugettext_lazy as _ %>
+<%! from django.utils.functional import Promise %>
+<%! from django.utils import six %>
+<%! from django.utils.functional import lazy %>
+<%! from datetime import date %>
```
Comment (amirtds) [inline lms/templates/theme-variables.html]: @bryanlandia I think this should be `<%! from datetime import datetime, date %>` We use `datetime.now()` to get current year https://github.com/appsembler/edx-theme-customers/blob/648708c85452717577b3e473ab57c4b9a0f4cb33/lms/templates/theme-variables.html#L166

**Review by amirtds — CHANGES_REQUESTED** [review]

**Review by amirtds — APPROVED** [review]
Thanks @bryanlandia It looks good. I checked quickly the staging server here https://learn.extension-dev.psu.edu/certificates/5cfe5a84c7074358adb2b516c544b9b3 and looks good

**Comment by bryanlandia** [comment]
I'm not merging yet since we don't have any approval yet from PSU and don't want these deployed to prod accidentally.

**Comment by bryanlandia** [comment]
@amirtds  I'm going to merge this in advance of probable production push Wednesday or Thursday of this week.  I rebased off of ginkgo/pennstate so it's got your latest changes.

**Pull request merged**